projects/1.20/assets/project-expansion/projectexpansion/lang/zh_cn.json

```
@@ -0,0 +1,585 @@
+	"block.projectexpansion.pink_emc_link": "粉色物质EMC链接器 [MK 5]",
+	"block.projectexpansion.purple_emc_link": "紫色物质EMC链接器 [MK 6]",
+	"block.projectexpansion.violet_emc_link": "深紫物质EMC链接器 [MK 7]",
+	"block.projectexpansion.blue_emc_link": "蓝色物质EMC链接器 [MK 8]",
+	"block.projectexpansion.cyan_emc_link": "青色物质EMC链接器 [MK 9]",
+	"block.projectexpansion.green_emc_link": "绿色物质EMC链接器 [MK 10]",
+	"block.projectexpansion.lime_emc_link": "浅绿物质EMC链接器 [MK 11]",
+	"block.projectexpansion.yellow_emc_link": "黄色物质EMC链接器 [MK 12]",
+	"block.projectexpansion.orange_emc_link": "橙色物质EMC链接器 [MK 13]",
+	"block.projectexpansion.white_emc_link": "白色物质EMC链接器 [MK 14]",
+	"block.projectexpansion.fading_emc_link": "渐隐物质EMC链接器 [MK 15]",
+	"block.projectexpansion.final_emc_link": "终极EMC链接器 [MK 16]",
+	"block.projectexpansion.magenta_fuel_block": "品红燃料块",
+	"block.projectexpansion.pink_fuel_block": "粉色燃料块",
+	"block.projectexpansion.purple_fuel_block": "紫色燃料块",
+	"block.projectexpansion.violet_fuel_block": "深紫燃料块",
+	"block.projectexpansion.blue_fuel_block": "蓝色燃料块",
+	"block.projectexpansion.cyan_fuel_block": "青色燃料块",
+	"block.projectexpansion.green_fuel_block": "绿色燃料块",
+	"block.projectexpansion.lime_fuel_block": "浅绿燃料块",
+	"block.projectexpansion.yellow_fuel_block": "黄色燃料块",
+	"block.projectexpansion.orange_fuel_block": "橙色燃料块",
+	"block.projectexpansion.white_fuel_block": "白色燃料块",
+	"block.projectexpansion.magenta_matter_block": "品红物质块",
+	"block.projectexpansion.pink_matter_block": "粉色物质块",
+	"block.projectexpansion.purple_matter_block": "紫色物质块",
+	"block.projectexpansion.violet_matter_block": "深紫物质块",
+	"block.projectexpansion.blue_matter_block": "蓝色物质块",
+	"block.projectexpansion.cyan_matter_block": "青色物质块",
+	"block.projectexpansion.green_matter_block": "绿色物质块",
+	"block.projectexpansion.lime_matter_block": "浅绿物质块",
+	"block.projectexpansion.yellow_matter_block": "黄色物质块",
+	"block.projectexpansion.orange_matter_block": "橙色物质块",
+	"block.projectexpansion.white_matter_block": "白色物质块",
+	"block.projectexpansion.fading_matter_block": "渐隐物质块",
+	"block.projectexpansion.arcane_table": "奥术转换桌",
+	"block.projectexpansion.transmutation_interface": "转换接口",
+	"block.projectexpansion.transmutation_interface.tooltip": "可与其他兼容模组（如AE2）搭配使用，使得你可以直接消耗EMC来使用已学习的物品。",
+	"block.projectexpansion.compact_sun": "压缩太阳",
+	"block.projectexpansion.compact_sun.tooltip": "一方之间，尽握太阳之力。",
+	"block.projectexpansion.compact_sun.tooltip2": "当放置在某些方块的上方或下方时，将其产出倍率提升%s倍。",
+	"block.projectexpansion.power_flower.tooltip": "每%s游戏刻生产EMC（每秒20刻）。",
+	"block.projectexpansion.power_flower.emc": "生产EMC：%s/s",
+	"block.projectexpansion.power_flower.stack_emc": "堆栈生产EMC：%s/s",
+	"block.projectexpansion.basic_power_flower": "基础能量盆栽 [MK 1]",
+	"block.projectexpansion.dark_power_flower": "暗物质能量盆栽 [MK 2]",
+	"block.projectexpansion.red_power_flower": "红物质能量盆栽 [MK 3]",
+	"block.projectexpansion.magenta_power_flower": "品红物质能量盆栽 [MK 4]",
+	"block.projectexpansion.pink_power_flower": "粉色物质能量盆栽 [MK 5]",
```

```suggestion
	"block.projectexpansion.pink_power_flower": "粉物质能量盆栽 [MK 5]",
```
同上，如“暗物质”、“红物质”，projectex化用了原版十六色作为物品的名字，请将颜色前缀作为物品名称的一部分
【要求】此类问题需要全文通查并修改

---

炼金箱子那块我可以理解，但是能量盆栽这一类用XX物质来合成的东西（因为有渐隐级别的物品在）应该是绑定XX物质的名称，所以不绑定原版十六色的名称吧？还是说例如“Lime Compressed Collector [MK 11]”这样的物品应该翻译为原版十六色的“压缩黄绿色收集器 [MK 11]”而并非projectex的“压缩浅绿物质收集器 [MK 11]”？（假设projectex的该物质译名为浅绿物质）。我没太能理解这块，还望解惑。

---

是绑定原有物质的名称，但是使用原版术语的同时不加“色”。
例如
* 原版：Lime→黄绿色
* 模组：Lime Matter→黄绿物质（而非浅绿物质、黄绿色物质）
* 模组：Lime Compressor

---

好的谢谢，明白了